### PR TITLE
chore(release): bump to 5.0.0-beta.6

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Bump to 5.0.0-beta.6

# Changelog

- bumped versions to 5.0.0-beta.6

# Review requests

-  Changelogs look reasonable?

# Risk assessment

low
****

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -22,17 +22,16 @@ This is beta software! You may experience unexpected crashes or missing features
 
 ## Bug Fixes
 
-Keep an eye on this space as we release new versions of the beta. See the [issue tracker][] for a full list of issues resolved by the v5 release.
+Keep an eye on this space as we release new versions of the beta. See the [issue tracker][] for a full list of issues resolved by the v6 release.
```
Comment (mcous) [inline api/release-notes.md]:
```suggestion
Keep an eye on this space as we release new versions of the beta. See the [issue tracker][] for a full list of issues resolved by the v5 release.
```

File: api/release-notes.md [hunk 2]
```diff
@@ -22,17 +22,16 @@ This is beta software! You may experience unexpected crashes or missing features
...
 
-[issue tracker]: https://github.com/Opentrons/opentrons/labels/fixed-in-v5
+[issue tracker]: https://github.com/Opentrons/opentrons/labels/fixed-in-v6
```
Comment (mcous) [inline api/release-notes.md]:
```suggestion
[issue tracker]: https://github.com/Opentrons/opentrons/labels/fixed-in-v5
```

File: app-shell/build/release-notes.md [hunk 3]
```diff
@@ -24,14 +24,7 @@ This is beta software! You may experience unexpected crashes or missing features
...
 
-Keep an eye on this space as we release new versions of the beta. See the [issue tracker][] for a full list of issues resolved by the v5 release.
-
-Since the last beta release we have fixed the following critical issues:
-
-- Fix an issue where missing metadata on commands caused a white screen when expanding protocol setup steps
-- Display labware offsets for labware on top of modules
-- Fix Labware Position Check summary screen offset data flickering
-- Fix labware offsets not rendering on deckmap after completing labware position check
+Keep an eye on this space as we release new versions of the beta. See the [issue tracker][] for a full list of issues resolved by the v6 release.
```
Comment (mcous) [inline app-shell/build/release-notes.md]:
```suggestion
Keep an eye on this space as we release new versions of the beta. See the [issue tracker][] for a full list of issues resolved by the v5 release.
```

**Review by mcous — COMMENTED** [review]
Looks like some find-and-replace issues

**Review by shlokamin — APPROVED** [review]
🚢

**Review by mcous — APPROVED** [review]

**Pull request merged**